test/OpenSpaceToolkit/Physics/Coordinate/Spherical/LLA.test.cpp

```diff
@@ -347,3 +347,126 @@ TEST_F(OpenSpaceToolkit_Physics_Coordinate_Spherical_LLA, Cartesian)
         EXPECT_ANY_THROW(LLA::Cartesian(Vector3d::Undefined(), Length::Undefined(), Real::Undefined()));
     }
 }
+
+TEST_F(OpenSpaceToolkit_Physics_Coordinate_Spherical_LLA, DistanceBetween)
```

Comment:
Shall we add a test for `distanceTo` as well?

Reply:
I'd also recommend splitting into two test `DistanceBetween_Spherical` and `DistanceBetween_WGS84`

Reply:
> Shall we add a test for distanceTo as well?

Oups, missed that! Will do